### PR TITLE
NDEV-2229 Make tests audit

### PR DESCRIPTION
**Pull request by gigimon**

### CODE REVIEW

File: docs/how_to_run.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+To run OpenZeppelin tests just use next command:
+```bash
+./clickfile.py run oz --network <your-stand> --jobs 8
+```
+
+
+## Run tests manual
+
+You can run all tests manual uif know which framework it uses. For example economy tests:
```
Comment (kristinaNikolaevaa) [inline docs/how_to_run.md]: manually if

File: docs/tests/audit/test_rpc_calls.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| TestRpcCalls::test_eth_get_balance                                      | Get sender balance with different state                      |           |
+| TestRpcCalls::test_eth_get_code                                         | Get code???                                                  |           |
+| TestRpcCalls::test_web3_client_version                                  | Get web3_client_version                                      |           |
+| TestRpcCalls::test_netversion                                           | Get net_version                                              |           |
+| TestRpcCalls::test_eth_get_transaction_count                            | Get sender nonce                                             |           |
+| TestRpcCalls::test_eth_send_raw_transaction                             | Basic check that sendRawTransaction work (send 1 neon)       |           |
+| TestRpcCalls::test_eth_send_raw_transaction                             | Basic check that sendRawTransaction work (send 1 neon)       |           |
```
Comment (kristinaNikolaevaa) [inline docs/tests/audit/test_rpc_calls.md]: test_eth_send_raw_transaction 2 times

File: docs/tests/audit/test_transfers.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+| TestTransfer::test_send_neon_from_one_account_to_another                    | Send 0, 0.1, 1, 1.1 neon from one account to another??                                    |                     |
+| TestTransfer::test_send_erc20_token_from_one_account_to_another             | Deploy erc20 contract and send 0, 1, 10, 100 tokens from one account to another??         |                     |
+| TestTransfer::test_send_spl_wrapped_token_from_one_account_to_another       | Deploy erc20spl contract and send 0, 1, 10, 100 tokens from one account to another??      |                     |
+| TestTransfer::test_send_more_than_exist_on_account_neon                     | Send more than exist in account \[11_000_501, 10_000_000.1\] and get an error??           |                     |
+| TestTransfer::test_send_more_than_exist_on_account_spl                      | Send erc20spl more than exist in account 1_000_000_000_000_000_000_000 and get an error?? |                     |
+| TestTransfer::test_send_more_than_exist_on_account_erc20                    | Send erc20 more than exist in account 100_000 and get an error??                          |                     |
```
Comment (kristinaNikolaevaa) [inline docs/tests/audit/test_transfers.md]: what do '??' means?

**Review by kristinaNikolaevaa — APPROVED** [review]

File: docs/tests/audit/test_transfers.md (hunk 3 above)
Comment (gigimon) [reply docs/tests/audit/test_transfers.md]: it was a mark to highlight this in issue ))  I will remove it

**Pull request merged**